internal/command/admin/deploy/docker/docker_down_test.go

```diff
@@ -66,6 +66,46 @@ func TestDockerDownRun(t *testing.T) {
 		require.Error(t, err)
 	})
 
+	t.Run("should return error when unable to find docker compose but local deployment is found", func(t *testing.T) {
```

Comment:
The message should probably be "[...] when unable to find docker-compose.yml" for better clarity (`docker compose` is technically the command, not the file).